srcCpp/RTIDDSImpl.cxx

```diff
@@ -934,7 +935,8 @@ class RTIPublisher : public IMessagingWriter
         return (unsigned int)status.pulled_sample_count;
     };
 
-    void wait_for_acknowledgments(const DDS_Duration_t & timeout){
+    void wait_for_acknowledgments(long sec, unsigned long nsec){
```

Comment:
Could you add a space between ")" and "{"

Reply:
Done